cli/test/e2e/package_repository_test.go

```diff
@@ -181,6 +186,46 @@ func TestPackageRepository(t *testing.T) {
 		require.Exactly(t, expectedOutputRows, output.Tables[0].Rows)
 	})
 
+	logger.Section("create a repository with secretRef", func() {
```

Comment:
Since, these are new tests let's add these two cases as the last ones, that way we can verify more obvious cases before these.